source/localizable/getting-started/quick-start.md

```diff
@@ -211,13 +211,6 @@ curly braces (`{{component}}`). We're going to tell our component:
    provide this route's `model` as the list of people.
 
 ```app/templates/scientists.hbs{-1,-2,-3,-4,-5,-6,-7,+8}
```

Comment:
This https://gist.github.com/spastorino/810b47638232affdcca8 is what makes styles look like 👆